### ISSUE TITLE
AAC 256K m4a audio gone after July 18, 2015 (OPUS too?)

### ISSUE THREAD

**Issue by BrianHath777** [inbasic/iaextractor#97]
not sure what's going on - previous YouTube vids that had the 256K audio option now not showing it - my "last update" msg was for July 18 for the downloader extension in Firefox ......

**Comment by wallrik** [comment]
This has been reported in issue #89 as well

**Comment by Metaphorce812** [comment]
I don't know why (maybe just to piss people off?), but YouTube recently stopped providing AAC 256kbps CBR DASH streams (scroll down and notice that iTag 141 has been discontinued):
https://en.wikipedia.org/wiki/YouTube#Quality_and_formats

This means that the highest quality AAC DASH audio is 128kbps CBR, which sucks. However 720p combined video+audio streams contain ~192kbps VBR AAC which is now the highest quality audio available anywhere on YouTube.
Would it be possible to extract that audio stream from the 720p video and combine it with the 1080p DASH video to get the highest quality video/audio in one file?

**Comment by augur42** [comment]
Quoted from issue #89 

> There is no reason why you couldn't download both the 720p192aac and 1080p dash and then mux the 192aac into the 1080p dash, just use ffmpeg.
In fact I just tried downloading the 1080p dash with automatic 128aac and the 720p192aac versions of a video, dragged them both into the Media Converter and Muxer (The split off Conversion Tools) and it output a 1080p192aac file. NB the audio stream muxed into the output is determined by the order the two files are selected for dropping into the window. I had to select the file with the audio stream I wanted first i.e. the 720p192aac then the 1080p128aac. It didn't matter the order for the video, no matter which I picked first the output was always 1080p.
